### PR TITLE
Masks C & D

### PR DESCRIPTION
**Pull request by zscoman**
Mask C is iNeurons for Sherry & Maria Clara; Mask D is iPSCs for Maria Clara. Both include plasma membrane as organelles.

### CODE REVIEW

File: infer_subc/organelles/cellmask.py
Comment (shanrhoads) [inline infer_subc/organelles/cellmask.py:126]: Why scale from 0-10 instead of using a 0-1 min-max, like min_max_intensity_normalization func. from infer_subc.core.img?

File: infer_subc/organelles/membrane.py
Comment (shanrhoads) [inline infer_subc/organelles/membrane.py:20]: Commented on this function in cellmask.py with something that may negate the need for this function. If it is needed, let's only keep it in one location. My suggestion for location is in infer_subc.core.img and can then be imported here and the other location as needed.

File: infer_subc/organelles/cellmask.py
Comment (shanrhoads) [inline infer_subc/organelles/cellmask.py:131]: Seems like this function is better worked out in the membrane.py file. Are they the same? If so, I think it should live only in the membrane.py location.

File: infer_subc/organelles/membrane.py
Comment (shanrhoads) [inline infer_subc/organelles/membrane.py:1]: All of this looks good. One additional to-do item is to write a "infer_plasmamembrane_fromlabel()" function that runs just like the workflow would, but from the code instead of the GUI.

File: notebooks/01_infer_nuclei_from-label.ipynb
Comment (shanrhoads) [inline notebooks/01_infer_nuclei_from-label.ipynb:1]: No mods needed in this file for this pull request - needs to be removed before request approval.

File: notebooks/02_infer_cellmask_from-composite_with-nuclei.ipynb
Comment (shanrhoads) [inline notebooks/02_infer_cellmask_from-composite_with-nuclei.ipynb:1]: No mods needed in this file for this pull request - needs to be removed before request approval.

File: notebooks/04_infer_neurites_soma_from-cellmask.ipynb
Comment (shanrhoads) [inline notebooks/04_infer_neurites_soma_from-cellmask.ipynb:1]: No mods needed in this file for this pull request - needs to be removed before request approval.

File: notebooks/05_infer_lysosome.ipynb
Comment (shanrhoads) [inline notebooks/05_infer_lysosome.ipynb:1]: No mods needed in this file for this pull request - needs to be removed before request approval.

File: notebooks/07_infer_golgi.ipynb
Comment (shanrhoads) [inline notebooks/07_infer_golgi.ipynb:1]: No mods needed in this file for this pull request - needs to be removed before request approval.

File: notebooks/08_infer_peroxisome.ipynb
Comment (shanrhoads) [inline notebooks/08_infer_peroxisome.ipynb:1]: No mods needed in this file for this pull request - needs to be removed before request approval.

File: notebooks/09_infer_ER.ipynb
Comment (shanrhoads) [inline notebooks/09_infer_ER.ipynb:1]: No mods needed in this file for this pull request - needs to be removed before request approval.

File: notebooks/10_infer_lipid_droplet.ipynb
Comment (shanrhoads) [inline notebooks/10_infer_lipid_droplet.ipynb:1]: No mods needed in this file for this pull request - needs to be removed before request approval.

**Review by shanrhoads — CHANGES_REQUESTED** [review]
The modifications in notebooks 1-10 are causing conflicts in the pull request. These changes need to be removed from the branch before I can prove this request.

The rest of the code looks pretty good, though. I only had a couple of comments, mostly about redundancy between files that seems unnecessary. Can you have a look at those comments in membrane.py and cellmask.py before submitting another pull request?

File: infer_subc/organelles/cellmask.py
Comment (zscoman) [reply infer_subc/organelles/cellmask.py:131]: yeah, it works better in the membrane.py file, that other one is an artifact from an earlier push to github and is no longer present in my code, I can go ahead and make github reflect that on monday

File: infer_subc/organelles/cellmask.py
Comment (zscoman) [reply infer_subc/organelles/cellmask.py:126]: removed edits in cellmask.py

File: infer_subc/organelles/membrane.py
Comment (zscoman) [reply infer_subc/organelles/membrane.py:20]: went ahead and removed this from the code and used minmaxintensity function

**Pull request closed without merging**
